test/helpers/envtest.go

```diff
@@ -75,7 +75,7 @@ func init() {
 	utilruntime.Must(infrav1.AddToScheme(scheme))
 
 	// Get the root of the current file to use in CRD paths.
-	_, filename, _, _ := goruntime.Caller(0) //nolint
+	_, filename, _, _ := goruntime.Caller(0) //nolint:dogsled
```

Comment:
Wondering if we can replace this call with something less dogsleddy - we have the same issue in core CAPI.

Reply:
maybe `os.Args[0]`?

Reply:
I'm not too worried about it. It's test code and as long as it works. If we replace it with something else, please let's make absolutely sure that it still works in all cases

Reply:
So maybe let's just do the same as suggested in: https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/pull/2294#discussion_r1308445183?